### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by divyap1**
Key actions have been added and tested. And everything else seems to work like it should, Can someone just check that what I've done hasn't changed the networking stuff?

### CODE REVIEW

File: src/controller/GameController.java [hunk 1]
```diff
@@ -38,14 +36,19 @@
...
-	private ServerController serverController;
+	private ServerController serverController;
+	private ActionController actionController;
 
 	private final boolean isHost;
 	private int playerCount;
 
 	/**
 	 * Delegates the creation of the MVC and then starts the game
+<<<<<<< HEAD
```
Comment (Recursively) [inline src/controller/GameController.java]: Need to remove this head commit comment

File: res/models/free_standing_whiteboard.obj [hunk 2]
```diff
@@ -0,0 +1,253 @@
+# Blender v2.75 (sub 0) OBJ File: 'free_standing_whiteboard.blend'
```
Comment (Recursively) [inline res/models/free_standing_whiteboard.obj]: Need to remove this as it's not correct at the moment

Comment (Recursively) [inline res/models/free_standing_whiteboard.obj]: The whole free_standing_whiteboard.obj model that is

Comment (divyap1) [inline res/models/free_standing_whiteboard.obj]: As in delete that file?

**Comment by Recursively** [comment]
Looks good to me

**Comment by Recursively** [comment]
Pulling

**Pull request merged**